Modules/Filtering/LabelMap/include/itkStatisticsLabelMapFilter.hxx

```diff
@@ -128,7 +128,7 @@ StatisticsLabelMapFilter<TImage, TFeatureImage>::ThreadedProcessLabelObject(Labe
 
     // increase the sums
     sum += v;
-    sum2 += std::pow(static_cast<double>(v), 2);
+    sum2 += Math::sqr(static_cast<double>(v));
```

Comment:
This one is arguably better left as it way, to be more similar to the next 2 lines... but I don't feel strongly about it.

Reply:
I see your point, thanks, but on the other hand, the next line (`sum3 += std::pow(static_cast<double>(v), 3)`) might as well use `Math::cube` instead  😸